README.md

```diff
@@ -20,10 +20,18 @@ import '@github/auto-check-element'
 </auto-check>
 ```
 
+```erb
+<auto-check src="/signup-check/username">
+  <input>
+  <input hidden data-csrf value="<%= authenticity_token_for("/signup-check/username") %>">
```

Comment:
Might be worth explicitly documenting why `data-csrf` is used as the selector, and not `name="csrf"`